src/Symfony/Component/Validator/Violation/ConstraintViolationBuilder.php

```diff
@@ -46,8 +46,9 @@ class ConstraintViolationBuilder implements ConstraintViolationBuilderInterface
 
     /**
      * @param TranslatorInterface $translator
+     * @param string              $message    The error message as a string or a stringable object
```

Comment:
please put it before $translator, to respect the declaration order

Reply:
Done.  Thanks for all the reviews.  This would be my first contribution to Symfony, so I appreciate the guidance.